portal-web/test/functional/com/liferay/portalweb/macros/json/util/JSONUtil2.macro

```diff
@@ -142,18 +142,25 @@ definition {
 
 	@summary = "returns ${userEmail}:${password} to be used in JSON calls ONLY"
 	macro formatJSONUser {
-		var defaultUserEmail = JSONUtil2.getDefaultUserEmail();
+		var defaultUserEmail = JSONUtil2.getDefaultUserEmail(specificURL = ${specificURL});
 		var defaultUserPassword = JSONUtil2.getDefaultUserPassword();
 
 		return "${defaultUserEmail}:${defaultUserPassword}";
 	}
 
 	@summary = "This will grab the default user email to be used in JSON or other omniadmin actions ONLY"
 	macro getDefaultUserEmail {
-		var customDefaultUserEmail = PropsUtil.get("custom.default.user.email");
+		if (isSet(specificURL)) {
```

Comment:
We dont need to nest the if statement, it should be If specificURL is set (now domain), then parse it and then use that domain, else if custom.default.user.email is set, we use that, otherwise we default to test@liferay.com. 

The reason why its better to do this is that we always want to keep any logic as simple as we possibly can and avoid nesting if statements as much as possible, especially if people expand on this macro in the future, otherwise it will become a real headache in the future (Sometimes there are like 3-4 nested if else statements and it becomes very difficult to read)